constructs/postgres.ts

```diff
@@ -57,7 +57,7 @@ export class PostgresDB extends Construct {
       identifier: `${name}`,
 
       engine: 'postgres',
-      engineVersion: '15.5',
```

Comment:
We could disable auto updates from AWS, but then we should make sure to often update ourselves.

Reply:
True, but I think it is fine to leave it like that ? Or do you think it would be more secure to be in charge of setting the engine version ourselves ? I feel like we will never update it ourselves ...